docs/DeveloperGuide.md

```diff
@@ -700,35 +700,110 @@ Given below are instructions to test the app manually.
 
 <div markdown="span" class="alert alert-info">:information_source: **Note:** These instructions only provide a starting point for testers to work on;
 testers are expected to do more *exploratory* testing.
-
 </div>
 
-### Launch and shutdown
-
-1. Initial launch
-
-   1. Download the jar file and copy into an empty folder
-
-   1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
 
-1. Saving window preferences
+### General
 
-   1. Resize the window to an optimum size. Move the window to a different location. Close the window.
+#### Launching TutorBuddy
+1. Download the jar file and copy into an empty folder
+1. Double-click the jar file Expected: Shows the GUI with a set of sample contacts. The window size may not be optimum.
```

Comment:
Maybe it would be better to bring the line from "Expected.." down 🤔

Reply:
`Shows the GUI with a set of sample contacts`: maybe sample students & tuition sessions instead of contacts?

Reply:
woopssii